index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+
+    <!-- google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300;400;500;700&family=Inter:wght@400;500;700&family=Montserrat:ital,wght@0,200;0,400;1,400&family=Roboto:ital,wght@0,300;0,400;0,500;1,400&display=swap" rel="stylesheet">
+</head>
+<body>
+  <!-- top header   -->
+  <div class="top-header">
+      <nav>
+        <h1 class="logo-art">Techmoves</h1>
+          <img src="images/humburger.png" class="humburger">
+      </nav>
```

Comment:
Great adding the `nav` tag 😉 but it should be wrapping the hamburger icon 🍔 and not also the logo. Your structure would be as below:
```
<header>
    <a>Logo</a>
    <nav>
       <img src="./hamburger' alt="Hamburger" />
    </nav>
</header>
```
I hope this helps 👀

Reply:
fixed